db/columns.py

```diff
@@ -2,7 +2,9 @@
 
 from db import connection as db_conn
 from db.deprecated.types.base import PostgresType
+from db.records import list_records_from_table
 
+from mathesar.utils.download_links import create_mash_for_uri
```

Comment:
This shouldn't be imported from the `db` layer. Imports go the other way.

So, I think this function should probably live in the `mathesar/utils/download_links.py` file.